srcpkgs/stoken/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'stoken'
+pkgname=stoken
+version=0.92
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkgconf"
+makedepends="libxml2-devel nettle-devel gtk+3-devel"
+depends="nettle libxml2"
+short_desc="RSA SecurID-compatible software token generator"
+maintainer="Andrew J. Hesford <ajh@sideband.org>"
+license="LGPL-2.1-or-later"
+homepage="https://sourceforge.net/p/stoken/wiki/Home/"
+distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.gz"
+checksum=aa2b481b058e4caf068f7e747a2dcf5772bcbf278a4f89bc9efcbf82bcc9ef5a
+
+stoken-gtk3_package() {
+	depends="${sourcepkg}>=${version}_${revision} gtk+3"
+	short_desc += " - GTK+3 frontend"
+
+	pkg_install() {
+		vmove usr/bin/stoken-gui
+		vmove usr/share/applications
+		vmove usr/share/pixmaps
+		vmove "usr/share/stoken/*.ui"
+
```

Comment:
please get rid of this newline

Reply:
This and the other change were made. Thanks!